simulation/halsim_gui/src/main/native/cpp/PWMSimGui.cpp

```diff
@@ -33,12 +33,14 @@ class PWMSimModel : public glass::PWMModel {
 
   glass::DoubleSource* GetSpeedData() override { return &m_speed; }
 
-  void SetSpeed(double val) override { HALSIM_SetPWMSpeed(m_index, val); }
+  void SetSpeed(double val) override {
+    HALSIM_SetPWMPulseMicrosecond(m_index, val);
+  }
```

Comment:
Naming of Speed functions should be updated to be PulseMicrosecond or similar.  Parameter/return types should also be changed from double to int32_t.  But most likely this just needs a major update to use the SimDevice pieces instead?  That can be a separate PR, we should just open an issue to track it.